### ISSUE TITLE
Feature requests.

### ISSUE THREAD

**Issue by williamlhunter** [NewSoupVi/The-Witness-Randomizer-for-Archipelago#26]
Apologies if this is the wrong place for this, but I have a couple feature requests:
1. I think it would be cool if the speed boosts were smaller permanent stacking sprint speed increases
2. It would be nice if i had the option to try skipped puzzles again to get the skips back.

I've got it working on Linux, no issues so far.  Thanks for your hard work!

**Comment by NewSoupVi** [comment]
I did see this but never answered it lol

1. This has been suggested a lot and I think it'd be cool, but there is some stuff to consider. This game is not like Stardew Valley where more speed is unambiguously good - Pressure Plates puzzles can be hard to solve at higher speed, and things like the Town Tower can become somewhat nauseating to navigate. The only way I would feel good about implementing this would be with a *toggle* that lets you turn on and off the added speed. It's not out of the question, but it's some extra effort hacking the game & making sure it feels good.
2. This is a cool idea, probably possible but quite hard to execute with the current code - The current code is very much based on not having to change back after skipping, making a lot of "destructive" changes to the panel. Maybe one day

Tl;dr: Both of these ideas are good, but they are also high effort to pull off well, so I wouldn't hold my breath atm
